### DISCUSSION TITLE
arm64 release for Ubuntu

### DISCUSSION THREAD

**Discussion by ilyaext** [The-OpenROAD-Project/OpenROAD#3779] [General]
I'm looking for an arm64 OpenROAD architecture release to install on Linux AWS EC2 arm64.
Can anyone give me a link or suggest how to compile?

**Comment by maliberty** [comment]
We don't test or release on arm64 but you can probably build from source.

**Comment by maliberty** [comment]
https://openroad.readthedocs.io/en/latest/user/Build.html

**Comment by maliberty** [comment]
https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildLocally.html for ORFS
